app/views/content/is-teaching-right-for-me/if-you-want-to-change-career.md

```diff
@@ -17,7 +17,7 @@ navigation_title: How do I change to a career in teaching?
 navigation_description: Use your experience to inspire students and bring your skills and expertise to the classroom.
 ---
 
-A career change into teaching allows you to use the skills and expertise you’ve already developed to inspire young people.
+Turn your passion into your profession with a career change into teaching. Not only can you use the skills and expertise you’ve already developed to inspire young people, teaching gives you the opportunity to grow and fulfill your own potential too.
```

Comment:
```suggestion
Turn your passion into your profession with a career change into teaching. Not only can you use the skills and expertise you’ve already developed to inspire young people, teaching gives you the opportunity to grow and fulfil your own potential too.
```
Think `fulfil` is more usual in UK? But one to check!

Otherwise changes look good to me!

Reply:
I had fulfil originally and it showed up as being the wrong spelling! So I added an l. I've double checked and it's slightly contradictory but The Guardian style guide, which I defer to after GOV.UK, has one l so I think you're right we should go with fulfil.